client/src/js/services/authinjector.js

```diff
@@ -1,10 +1,14 @@
 angular.module('bhima.services')
-.factory('AuthInjectorFactory', ['$rootScope','$q',  '$location', 'SessionService', AuthInjector]);
+.factory('AuthInjectorFactory', AuthInjectorFactory);
```

Comment:
This is to make an $http interceptor, which must return an object. A service would also work, I am pretty sure, but have not tested it.